src/business-logic/services/tv2-inews-ingest-service.ts

```diff
@@ -0,0 +1,12 @@
+import { IngestService } from './interfaces/ingest-service'
+import {HttpRequestParameters, HttpService} from './interfaces/http-service'
+
+export class Tv2InewsIngestService implements IngestService {
+  constructor(private readonly httpService: HttpService) {}
+  public async reloadIngestData(rundownName: string): Promise<void> {
+    const domain = process.env.INEWS_DOMAIN ?? 'localhost:3007'
```

Comment:
Magic string.
You could extract it into a variable called something liked `const DEFAULT_URL: string = 'localhost:3007'`

Reply:
Moved to constant!